src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfUploadService.java

```diff
@@ -72,11 +77,11 @@ private static String createPdfDocumentNameFromCaseData(CaseData caseData,
         if (isNullOrEmpty(claimantLastName)) {
             claimantLastName = userInfo.getFamilyName();
         }
-        return "ET1 - "
+        return documentType + "-"
             + sanitizePartyName(claimantFirstName)
-            + " "
+            + "_"
```

Comment:
Any reason why we are changing this? It used to be underscores before but was then changed to spaces

Reply:
Just to have file names without spaces. Generally it is accepted as a good practice to not to have spaces in file names. (https://superuser.com/questions/29111/what-technical-reasons-exist-for-not-using-space-characters-in-file-names) Normally also _ (underscores) are not advised to use because those file names may cause some conflicts in the command prompts. If there is a reason to use space, I can revert it back to space.

Reply:
Updated as  
return documentType + StringUtils.SPACE + STRING_DASH + StringUtils.SPACE
            + sanitizePartyName(claimantFirstName)
            + StringUtils.SPACE + STRING_DASH + StringUtils.SPACE
            + sanitizePartyName(claimantLastName)
            + (ENGLISH_LANGUAGE.equals(documentLanguage)
            ? ""
            : StringUtils.SPACE + STRING_DASH + StringUtils.SPACE + documentLanguage)
            + ".pdf";